### ISSUE TITLE
[Pomodoro] remove menu bar icon when not in use

### ISSUE THREAD

**Issue by Ling01234** [raycast/extensions#15945]
### Extension

https://www.raycast.com/asubbotin/pomodoro

### Description

Currently, when the extension is not in use, it's just filled with dashes in the menu bar. 

I would like to request an update or feature where when the extension is not in use, the icon is not in the menu bar anymore, and when it is in use, it shows up in the menu bar.

**Comment by LitoMore** [comment]
You can simply disable the menu bar icon from settings.

**Comment by Ling01234** [comment]
But then when i start a session, it's not in the menu bar anymore. Is it possible I'm misconfiguring it? If so, apologies for opening the issue

**Comment by mikikiv** [comment]
I think we could define a few options for when there is no active session and timer in menu is enabled. "--:--" (default), null (or none)

Not sure if there are any others that make sense or if it should be a text box rather than select even

**Comment by Ling01234** [comment]
Yeah that would be great to have that option. It would make the menu bar a lot cleaner in my opinion if we can choose the setting to none. Otherwise, it takes quite a bit of space in the menu bar and doesn't look great with the "--:--". 

I would love that feature if possible!
